src/open-api/utils/dereference.ts

```diff
@@ -0,0 +1,40 @@
+import { pointerToPath } from '@stoplight/json'
+
+/**
+ * TODO: Support remote references.
```

Comment:
I think this is fine for a first iteration. Remote reference can also easily open a can of worms, e.g. from which relative path would it load the remote files etc?